### DISCUSSION TITLE
Bulk inferences from glow-tts checkpoint?

### DISCUSSION THREAD

**Discussion by patdflynn** [coqui-ai/TTS#940] [General Q&A]
Hi there,

I'm new and a bit confused. There are examples of how to run inferences from CLI or to pull a model from torchhub, but I've not been able to find any on how to run bulk inferences from an existing checkpoint. 

My understanding is that something like the following is required:
1. Load Glow-tts and config (use training config?)
2. Load  Glow-tts checkpoint
3. Load a neural vocoder and config
4. Load neural vocoder checkpoint
5. Iteratively use decoder_inference and pass the result to the vocoder

Would anyone be willing to share a worked example?

**Comment by erogol** [comment]
Welcome to 🐸

What do you mean by "bulk inference"? Batched inference or just calling the models in Python ?

**Reply by patdflynn** [reply]
Thank you,

I suppose I'm talking about batched inference but I'd like to call it from a notebook using one of my own checkpoints

**Reply by erogol** [reply]
Mode is capable of batch inference but you need to implement it manually.

You can use Synthesizer class as your starting point.
